hypothesis-python/src/hypothesis/strategies/_internal/core.py

```diff
@@ -1677,15 +1687,24 @@ def complex_numbers(
             f"Cannot have allow_nan={allow_nan!r}, min_magnitude={min_magnitude!r} "
             f"max_magnitude={max_magnitude!r}"
         )
-
```

Comment:
nit: try avoid changes like this to keep the diff smaller

Reply:
I tried to group them in a more meaningful way, but yep, unerstood.